src/txn/txn_rollback_to_stable.c

```diff
@@ -818,6 +818,40 @@ __rollback_abort_newer_updates(
     return (ret);
 }
 
+/*
+ * __wt_rts_page_skip --
+ *     Skip if rts doesn't requires to read this page.
+ */
+int
```

Comment:
Why not just return a bool in this function?

Reply:
The `__wt_rts_page_skip` function is passed as a function pointer to the tree walk function. This must follow same signature of the function pointer, so it cannot be changed.